### PR TITLE
Python 3 support in sunpy.data

### PR DESCRIPTION
**Pull request by Cadair**
This one was easy.

This should be merged after #1569

### CODE REVIEW

**Comment by Cadair** [comment]
@nabobalis review this one as well please.

**Comment by Cadair** [comment]
Though you might want to wait till #1569 is in so the diff is clean.

**Comment by Cadair** [comment]
@dpshelio @nabobalis Next up this one.

**Comment by nabobalis** [comment]
I can't see any obvious issues. :+1:

File: continuous-integration/python3.sh [hunk 1]
```diff
@@ -3,3 +3,6 @@
 # This script runs a passing subset of tests under Python 3.
 
 python setup.py install
+python -c "import sunpy.data"
+python -c "import sunpy.data; sunpy.data.download_sample_data()"
+python -c "import sunpy.data.sample"
```
Comment (dpshelio) [inline continuous-integration/python3.sh]: why the three commands?

Comment (Cadair) [inline continuous-integration/python3.sh]: just to make is cleaner and easier to see which one was broken. This script is only temporary until the whole test suite works under 3.x

**Comment by dpshelio** [comment]
:+1:

**Pull request merged**